frontend/packages/policy-editor/src/components/PolicyEditor.tsx

```diff
@@ -58,24 +60,26 @@ interface Props {
  * @property {PolicyBackendType}[policy] - The policy to edit
  * @property {PolicyActionType[]}[actions] - A list of actions that can be used in the rules
  * @property {PolicySubjectType[]}[subjects] - A list of subjects that be can selected in the rules
- * @property {string}[resourceType] - The type of the resource so that it can fill the autogenerated field for the first resource
  * @property {string}[resourceId] - The ID of the resource so that it can fill the autogenerated field for the first resource
  * @property {function}[onSave] - Function that saves the policy
  * @property {boolean}[showAllErrors] - Flag to decide if errors should be shown or not
+ * @property {policyEditorUsageType}[usageType] - The usage type of the policy editor
  *
  * @returns {React.ReactNode} - The rendered component
  */
 export const PolicyEditor = ({
   policy,
   actions,
   subjects,
-  resourceType,
   resourceId,
   onSave,
   showAllErrors,
+  usageType,
 }: Props): React.ReactNode => {
   // TODO - translation
```

Comment:
It's already a task in the same issue as this PR. Just thought it would be better to separate the PRs so smaller mistakes are easier to spot. 
https://github.com/Altinn/altinn-studio/issues/10746